jonprl-mode.el

```diff
@@ -417,6 +433,7 @@ Invokes `jonprl-mode-hook'."
   (setq-local comment-start "|||")
   (jonprl-update-available-tactics)
   (setq-local font-lock-defaults (jonprl-font-lock-defaults))
+  (setq-local prettify-symbols-alist jonprl-pretty-symbols)
```

Comment:
Can this be moved to a hook that's enabled by default? I'd like to make it possible to override these, and that's a lot easier if there's just a hook to remove.